server/infra/database/BaseRepository.spec.ts

```diff
@@ -69,6 +69,43 @@ describe('BaseRepository', () => {
       expect(result[0]).toHaveProperty('id', 1)
     })
 
+    it('getByFiliter with order', async () => {
+      tracker.uninstall()
+      tracker.install()
+      tracker.on('query', (query) => {
+        expect(query.sql).toMatch('select \* from "testTable" where \("name" \= $1\) order by "id" desc'
```

Comment:
Nice, I thought you don't need to use the backslash to escape the star, unless you are typing regular express.